aiida/cmdline/commands/cmd_daemon.py

```diff
@@ -27,35 +27,46 @@
 from aiida.manage.configuration import get_config
 
 
+def validate_positive_non_zero_integer(ctx, param, value):  # pylint: disable=unused-argument,invalid-name
+    """Validate that `value` is a positive integer."""
+    if not isinstance(value, int):
+        raise click.BadParameter('{} is not an integer'.format(value))
+
+    if value <= 0:
+        raise click.BadParameter('{} is not a positive non-zero integer'.format(value))
+
+    return value
+
+
 @verdi.group('daemon')
 def verdi_daemon():
     """Inspect and manage the daemon."""
 
 
 @verdi_daemon.command()
 @click.option('--foreground', is_flag=True, help='Run in foreground.')
+@click.argument('number', default=1, type=int, callback=validate_positive_non_zero_integer)
```

Comment:
Could this be an option instead?
The command currently is `verdi start-circus 12` and I think `verdi start-circus --nworkers 12` (or similar) would be more self-explanatory.

Reply:
One never calls `verdi daemon start-circus` it is a hidden command. But it would be `verdi daemon start [NUMBER]`. I chose the argument because that is inline with the already existing commands `verdi daemon incr/decr [NUMBER]`.